doc/source/performance/aio_page_requests.rst

```diff
@@ -0,0 +1,44 @@
+.. _aio_page_requests:
+
+=======================================
+Multiple page asynchronous I/O requests
+=======================================
+
+I/O unit size in |InnoDB| is only one page, even if doing read ahead. 16KB
+I/O unit size is too small for sequential reads, and much less efficient than
+larger I/O unit size.
+
+|InnoDB| uses Linux Native I/O (``aio``) by default. By submitting multiple
+consecutive 16KB read requests at once, Linux internally can merge requests and
+reads can be done more efficiently.
+
+On modern HDD RAID 1+0 environment, more than 1000MB/s disk reads can be
```

Comment:
Where is this taken from? Please attach attribution to this claim. How old is "modern" here?